Source/Core/Common/Arm64Emitter.cpp

```diff
@@ -1434,23 +1429,23 @@ void ARM64XEmitter::BFI(ARM64Reg Rd, ARM64Reg Rn, u32 lsb, u32 width)
 {
   u32 size = Is64Bit(Rn) ? 64 : 32;
   ASSERT_MSG(DYNA_REC, lsb < size && width >= 1 && width <= size - lsb,
-             "%s passed lsb %d and width %d which is greater than the register size!", __func__,
+             "{} passed lsb {} and width {} which is greater than the register size!", __func__,
```

Comment:
Any particular reason why you removed the use of `__func__` earlier in the file but stopped here?

Reply:
I think it was partially getting distracted partway through, and partly because this message is one where it would need to be reworded more than just removing the function name followed by a colon.  I could get rid of it here too, but the message would need to be altered a bit.